tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -269,7 +269,7 @@ def test_root_event(self):
             child_event_id = child_event.event_id
             assert child_event_id in events
             event = events[child_event_id]
-            assert event["generation"] is None
+            assert event["generation"] == 1
```

Comment:
Why was this `None` before?

Reply:
Mistake on my part, when we had a direct line to the root on the quick trace we weren't getting child generations